framework/client/releasing.rst

```diff
@@ -0,0 +1,80 @@
+Releasing
+=================
+
+New Feature
+-----------
+New features will be developed in separate feature/pull-request branches. As soon as a feature is ready, a pull request is opened to rebase the feature into the master branch.
+After a successful rebase the feature branch can be deleted. Only self-contained features should be rebased to master to keep master stable and packages always ready to be published.
+
+A packages will be released on master only. For more information on how to publish a package see the tocco-clients README_.
+
+.. _README: https://github.com/tocco/tocco-client
+
+Semantic Releases
+~~~~~~~~~~~~~~~~~
+A package version consists of three numbers MAJOR.MINOR.PATCH.
+
+Increment as follows:
+
+====== ============================
+MAJOR  Will be zero for the moment
+MINOR  If new features are added
+PATCH  For bug fixes
+====== ============================
+
+
+Maintaining old versions
+-------------------------
+Older nice versions are using older releases of client packages. If a critical bug is found in an older package we need to fix it in that version. It's not possible to just
```

Comment:
Especially in english texts I'd write "Nice" instead of "nice" for clarity :) (everywhere in this document)